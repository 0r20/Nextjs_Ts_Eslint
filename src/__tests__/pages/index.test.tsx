import React from 'react';
import { mount, ReactWrapper } from 'enzyme';
import { act } from 'react-dom/test-utils';
import Index from '../../../pages/index';

describe('Pages', () => {
  describe('Index', () => {
    let TestIndexWrapper: ReactWrapper;
    it('should render without throwing an error', () => {
      act(() => {
        TestIndexWrapper = mount(<Index />);
      });
      expect(TestIndexWrapper.find('span').text()).toBe('This is index');
    });
  });
});
