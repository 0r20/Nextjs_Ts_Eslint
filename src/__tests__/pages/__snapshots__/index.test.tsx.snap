// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage should render without throwing an error 1`] = `
<IndexPage
  test="test"
>
  <main>
    <p>
      Props from _app.tsx: {"test":"test"}
    </p>
    <p>
      page process env: TEST_TEST_PAGE_PROP
    </p>
    <br />
    <PrefixedLink
      href="/index/1"
    >
      <Link
        as="/index/1"
        href="/index/1"
      >
        <a
          href="/index/1"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          index/1
        </a>
      </Link>
    </PrefixedLink>
  </main>
</IndexPage>
`;
